commcare_connect/opportunity/visit_import.py

```diff
@@ -112,41 +114,39 @@ def bulk_update_visit_status(opportunity: Opportunity, file: UploadedFile) -> Vi
 
 
 def _bulk_update_visit_status(opportunity: Opportunity, dataset: Dataset):
-    status_by_visit_id, reasons_by_visit_id = get_status_by_visit_id(dataset)
```

Comment:
is this old function still needed?

Reply:
Removed this function and fixed the tests that use this. [737592c](https://github.com/dimagi/commcare-connect/pull/383/commits/737592c1941610838a0c164924504d85b7511dcc)